tests/e2e/specs/wcpay/merchant/merchant-orders-full-refund.spec.js

```diff
@@ -39,19 +39,18 @@ describe( 'Order > Full refund', () => {
 		);
 		orderId = await orderIdField.evaluate( ( el ) => el.innerText );
 
+		// Get the order total so we can verify the refund amount
+		const orderTotalField = await page.$(
+			'.woocommerce-order-overview__total .woocommerce-Price-amount'
```

Comment:
Can we get this class name from a constant? Something like `ORDER_RECEIVED_AMOUNT_IDENTIFIER`. That way in case the test starts failing at this point, the variable name could be a hint to check whether something has changed there and save some debugging time.

Reply:
This is a great idea. I've implemented this in 5271335 for the order ID and amount selectors.